rest-api/src/main/java/gov/cms/qpp/conversion/api/services/DbServiceImpl.java

```diff
@@ -31,11 +32,14 @@
 	private static final int LIMIT = 3;
 	public static final String START_OF_UNALLOWED_CONVERSION_TIME = "2018-01-02T04:59:59.999Z";
 
-	@Inject
-	private DynamoDBMapper mapper;
+	private final Optional<DynamoDBMapper> mapper;
```

Comment:
Why did we make this an `Optional`?

Reply:
We return null for the provider when no audit is on, which in later versions of our dependencies causes exceptions. We can still return null, we just have to make the dependency an optional